index.js

```diff
@@ -427,6 +446,31 @@ class Jarvis {
       return true;
     }
   }
+
+  /**
+   * Load a provided env file
+   * Save env variables as constants
+   */
+  async _loadEnv(envFile) {
+    let fileContent;
+    try {
+      fileContent = parseScript(envFile);
+    } catch (error) {
+      throw new Error('Could not read env file from specified location!');
+    }
+    if (fileContent[0] === "in this context" && fileContent[fileContent.length - 1] === "end") {
+      for (let line of fileContent) {
+        line = line.trim();
+        const matches = line.match(/(.+) is ['"](.+)['"]/i);
+        if (matches) {
+          const [, key, value] = matches;
+          this.environmentVariables[key] = value;
+        }
+      }
+    } else {
+      return "Invalid syntax in env file!"
```

Comment:
don't mix returns and throws to signal errors.
also, this return value is undocumented in the function comment.